doc/cookbook/source/examples/evaluation/cross_validation_pipeline.rst

```diff
@@ -0,0 +1,34 @@
+============================
+Cross Validation on Pipeline
+============================
+
+In this example, we illustrate how to use cross-validation with :sgclass:`CPipeline`.
+
+For more information on cross-validation, check :doc:`./cross_validation`.
+
+-------
+Example
+-------
+We'll use as example a binary classification problem solvable by a pipeline consisted of a transformer :sgclass:`CPruneVarSubMean` and a machine :sgclass:`CLibLinear`.
+
+Imagine we have files with training data. We create :sgclass:`CDenseFeatures` (here 64 bit floats aka RealFeatures) as
+
+.. sgexample:: cross_validation_pipeline:create_features
+
+
+We use :sgclass:`CPruneVarSubMean` to normalize the features and then use :sgclass:`CLibLinear` for classification.
+The transformer and the machine are chained as a :sgclass:`CPipeline`.
```

Comment:
We create a Cpipeline, and chain the transformer and the classifier.

Reply:
actually, I would mention that "we first chain all transformers, and then finalize the pipeline with the classifier" (since you use "then")